infra/src/uber-charts/islandis.ts

```diff
@@ -249,9 +250,10 @@ export const FeatureDeploymentServices: ServiceBuilder<any>[] = []
 
 // Services that are included in some environment above but should be excluded from feature deployments
 export const ExcludedFeatureDeploymentServices: ServiceBuilder<any>[] = [
-  userNotificationService,
-  userNotificationWorkerService,
-  userNotificationCleanupWorkerService,
+  // REMEMBER TO UNCOMMENT THESE LINES BEFORE MERGING ON MAIN
```

Comment:
Friendly reminder to uncomment these lines since the comment says so 🙃

Reply:
haha, yeah it will be my last work before merging, i'm still using these in my feature deployment 😸